helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -375,13 +375,15 @@ public void removeDeploymentGroup(final String name) throws DeploymentGroupDoesN
     final ZooKeeperClient client = provider.get("removeDeploymentGroup");
     try {
       client.ensurePath(Paths.configDeploymentGroups());
-      client.delete(Paths.configDeploymentGroup(name));
+
```

Comment:
I am very confused. It just looks like you're reordering a line. What problem does this fix and how?